### PR TITLE
Create snapshot test for `adminTable` and `updateUserForm` components

### PR DESCRIPTION
**Pull request by Code-Victor**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Tackling #309

### CODE REVIEW

File: __tests__/components/adminTable.test.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        id: 3,
+        name: "Ayomide onifade",
+        email: "Jangulabi@gmail.com",
+        role: "TEACHER"
+    },
+]
+
+describe('AdminTable rendering correctly', () => {
+  it('renders correctly', () => {
```
Comment (utsab) [inline __tests__/components/adminTable.test.jsx:44]: Can you write a more descriptive name for this test, something more specific than "renders correctly"?  For example, "renders X rows and Y columns"

**Review by utsab — CHANGES_REQUESTED** [review]

File: __tests__/components/adminTable.test.jsx (hunk 1 above)
Comment (Code-Victor) [reply __tests__/components/adminTable.test.jsx:44]: Noted, I'll make some changes.

Comment (Code-Victor) [reply __tests__/components/adminTable.test.jsx:44]: I just updated the test name and made it more descriptive.

**Comment by lloydchang** [comment]
@Code-Victor Per our conversation, here are steps for your consideration:
1. "This branch is out-of-date with the base branch" — which you resolved during our meeting
2. You can `@` Utsab, such as `@utsab`, to re-reviewing your pull request  
3. Creating a commit with multiple authors at https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors
4. GitHub workflow: Try if multiple team members should work in the same or separate repositories; it depends.

**Comment by lloydchang** [comment]
@Code-Victor There are missing space characters in:
```
Co-authored-by: Alade Christopher<Aladechristoph@gmail.com>
Co-authored-by: Ayomide Onifade<onifadeayomide328@gmail.com>
```
You need to add a space character ` ` before the `<` character.

File: __tests__/components/__snapshots__/updateUserForm.test.jsx.snap
Comment (lloydchang) [inline __tests__/components/__snapshots__/updateUserForm.test.jsx.snap:1]: @Code-Victor There are missing space characters in:
```
Co-authored-by: Alade Christopher<Aladechristoph@gmail.com>
Co-authored-by: Ayomide Onifade<onifadeayomide328@gmail.com>
```
You need to add a space character ` ` before the `<` character.

Comment (lloydchang) [reply __tests__/components/__snapshots__/updateUserForm.test.jsx.snap:1]: @Code-Victor lgtm; the new commit says "3 people committed":
Screen Shot 2023-07-26 at 7 43 11 AM

**Comment by lloydchang** [comment]
@Code-Victor lgtm; the new commit says "3 people committed":
Screen Shot 2023-07-26 at 7 43 11 AM

**Comment by lloydchang** [comment]
Please contact @GuillermoFloresV if you have further questions about this. Thanks!

**Review by utsab — APPROVED** [review]

**Review by utsab — APPROVED** [review]

**Pull request merged**